ExchangeSharp/API/Exchanges/Bl3p/ExchangeBl3pAPI.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace ExchangeSharp.API.Exchanges.Bl3p
+{
+	public sealed class ExchangeBl3pAPI : ExchangeAPI
```

Comment:
From their site, the capitalization is BL3P. Conflicts slightly w/ C# capitalization guidelines, but I've been changing the exchange capitalization to match their official titles.

Reply:
Fixed in 84e1a15